test/forms/inputs/checkbox-group.test.js

```diff
@@ -118,10 +121,86 @@ test('CheckboxGroup passes down props to children', () => {
   }
   render(<CheckboxGroup {...props} />)
 
-  const checkboxGroup = screen.getByRole('group', { name: 'Test Group' })
+  const checkboxGroup = screen.getByRole('group', { name: formattedName })
   const checkbox = screen.getByRole('checkbox')
 
   expect(checkboxGroup).toHaveClass('custom-group-class')
   expect(checkbox).not.toHaveClass('custom-group-class')
   expect(checkbox).toHaveClass('custom-input-class')
 })
+
+test('CheckboxGroup with dropdown = true adds value to array when unselected option clicked', async () => {
+  const user = userEvent.setup()
+
+  render(<WrappedCheckboxGroup dropdown={true} />)
+
+  const select = screen.getByRole('button')
+  await user.click(select)
+
+  const firstCheckbox = screen.getByLabelText('First Option')
+  await user.click(firstCheckbox)
+
+  const thirdCheckbox = screen.getByLabelText('Third Option')
+  await user.click(thirdCheckbox)
+
+  expect(firstCheckbox).toBeChecked()
+  expect(thirdCheckbox).toBeChecked()
+
+  const selectValueLabel = screen.getByText('3, 1')
+  expect(selectValueLabel).toBeInTheDocument()
+})
+
+test('CheckboxGroup with dropdown = true removes value from array when selected option clicked', async () => {
+  const user = userEvent.setup()
+
+  render(<WrappedCheckboxGroup value={['1']} dropdown={true} />)
+
+  const select = screen.getByRole('button')
+  await user.click(select)
+
+  const firstCheckbox = screen.getByLabelText('First Option')
+  await user.click(firstCheckbox)
+
+  expect(firstCheckbox).not.toBeChecked()
+
+  const selectValueLabel = screen.getByText('None')
+  expect(selectValueLabel).toBeInTheDocument()
+})
+
+test('CheckboxGroup with dropdown = true sets menu no longer active when clicked outside', async () => {
+  const user = userEvent.setup()
+
+  render(<WrappedCheckboxGroup value={['1']} dropdown={true} />)
+
+  const select = screen.getByRole('button')
+  await user.click(select)
+
+  expect(select.nextSibling).toHaveClass('options', 'is-active')
+
+  const fieldset = screen.getAllByRole('group').at(0)
+
+  user.click(fieldset)
+  await waitFor(() => {
+    expect(select.nextSibling).not.toHaveClass('is-active')
+  })
+})
+
+test('when no custom required indicator provided, do not show required indicator', () => {
```

Comment:
There is no outer `describe` block in this test file, so this test case (and the remaining) should be prefixed with `CheckboxGroup`